optaplanner-benchmark/src/test/resources/org/optaplanner/benchmark/config/testBenchmarkConfigWithNamespace.xml

```diff
@@ -32,7 +32,7 @@
       <solutionClass>org.optaplanner.core.impl.testdata.domain.TestdataSolution</solutionClass>
       <entityClass>org.optaplanner.core.impl.testdata.domain.TestdataEntity</entityClass>
       <scoreDirectorFactory>
-        <scoreDrl>org/optaplanner/core/api/solver/testdataConstraints.drl</scoreDrl>
```

Comment:
As far as I can tell, this file has been removed ages ago, but the tests somehow still used it.
Clearly, these tests should be using a ConstraintProvider too.